datasets/wiki_qa_ar/wiki_qa_ar.py

```diff
@@ -0,0 +1,119 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Arabic Wiki Question Answering corpus."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{YangYihMeek:EMNLP2015:WikiQA,
+       author = {{Yi}, Yang and {Wen-tau},  Yih and {Christopher} Meek},
+        title = "{WikiQA: A Challenge Dataset for Open-Domain Question Answering}",
+      journal = {Association for Computational Linguistics},
+         year = 2015,
+          doi = {10.18653/v1/D15-1237},
+        pages = {2013–2018},
+}
+"""
+
+_DESCRIPTION = """\
+Arabic Version of WikiQA by automatic automatic machine translators \
+and crowdsourced the selection of the best one to be incorporated into the corpus
+"""
+
+_URL = "https://raw.githubusercontent.com/qcri/WikiQAar/master/"
+_URL_FILES = {
+    "train": _URL + "WikiQAar-train.tsv",
+    "dev": _URL + "WikiQAar-dev.tsv",
+    "test": _URL + "WikiQAar-test.tsv",
+}
+
+
+class WikiQaArConfig(datasets.BuilderConfig):
+    """BuilderConfig for WikiQaAr."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for WikiQaAr.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(WikiQaArConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+
+
+class WikiQaAr(datasets.GeneratorBasedBuilder):
+    """WikiQaAr dataset."""
+
+    BUILDER_CONFIGS = [
+        WikiQaArConfig(
+            name="plain_text",
+            description="Plain text",
+        )
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "question_id": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "answer_id": datasets.Value("string"),
+                    "answer": datasets.Value("string"),
+                    "label": datasets.features.ClassLabel(num_classes=2),
```

Comment:
do the labels have names that we can use here ?

Reply:
Not that I am aware of. I think it is 1 if the sentence is an answer to the question or 0 otherwise.

Reply:
from the Github repo: "We make this dataset publicly available for the community to explore models such as cross-language question answering and answer triggering across languages; i.e. deciding if an answer exists in the same language as the question and ---if not--- searching in the other language."

Can you update the `Data Fields` in the README to reflect that?

Reply:
The dataset was translated from English and it can be used for bi-lingual tasks. I didn't include the English data because I think it is already added in `wiki_qa` https://huggingface.co/datasets/wiki_qa.

Reply:
Yes! My understanding was that the `label` indicates whether the answer can be found in Arabic Wikipedia or not: if that is indeed the case, we should say it in the README.md :)

Reply:
I already made a commit with the updated README.md.